### ISSUE TITLE
inline emojis should be crossposted

### ISSUE THREAD

**Issue by pogmommy** [pogmom/TwitToMast#8]
emojis currently are not crossposted to mastodon, because twitter displays them as images rather than emoji characters. The alt text is the correct emoji however, so replacing the image with it's alttext should be enough to implement this
